// @tsparticles/preset-snow@3.2.0/umd/options.js
(function (factory) {
    if (typeof module === "object" && typeof module.exports === "object") {
        var v = factory(require, exports);
        if (v !== undefined) module.exports = v;
    }
    else if (typeof define === "function" && define.amd) {
        define(["require", "exports", "@tsparticles/engine"], factory);
    }
})(function (require, exports) {
    "use strict";
    Object.defineProperty(exports, "__esModule", { value: true });
    exports.options = void 0;
    const engine_1 = require("@tsparticles/engine");
    exports.options = {
        background: {
            color: "#333",
        },
        particles: {
            number: {
                value: 100,
            },
            move: {
                direction: engine_1.MoveDirection.bottom,
                enable: true,
                random: false,
                straight: false,
            },
            opacity: {
                value: { min: 0.1, max: 0.5 },
            },
            size: {
                value: { min: 1, max: 10 },
            },
            wobble: {
                distance: 20,
                enable: true,
                speed: {
                    min: -5,
                    max: 5,
                },
            },
        },
    };
});
