// @tsparticles/preset-bubbles@3.2.0/cjs/options.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.options = void 0;
const engine_1 = require("@tsparticles/engine");
exports.options = {
    fpsLimit: 120,
    particles: {
        color: {
            value: "random",
        },
        shape: {
            type: "circle",
        },
        opacity: {
            value: 0.3,
        },
        size: {
            value: { min: 5, max: 10 },
        },
        move: {
            angle: {
                offset: 0,
                value: 30,
            },
            enable: true,
            speed: 15,
            direction: engine_1.MoveDirection.top,
            random: false,
            straight: false,
            outModes: {
                default: engine_1.OutMode.destroy,
            },
        },
    },
    detectRetina: true,
    background: {
        color: "#fff",
    },
    emitters: [
        {
            direction: engine_1.MoveDirection.top,
            position: {
                y: 100,
            },
            life: {
                duration: 3,
                delay: 5,
                count: 0,
            },
        },
    ],
};
